### PR TITLE
Feature 2

### PR DESCRIPTION
**Pull request by turkishhoneybadger**
First "create pull request"

### CODE REVIEW

File: fature2/one-more-file.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+One more file has been added.
```
Comment (bkamberoglu) [inline fature2/one-more-file.txt:1]: Nice try!

Comment (turkishhoneybadger) [reply fature2/one-more-file.txt:1]: Thanks for this comment.

**Review by bkamberoglu — APPROVED** [review]
Good Job!

**Comment by turkishhoneybadger** [comment]
Done

**Pull request closed without merging**